setup.py

```diff
@@ -44,6 +44,7 @@ def _make_required_install_packages():
   return [
       'absl-py>=0.9,<0.13',
       'apache-beam[gcp]>=2.33,<3',
+      'future==0.18.2',
```

Comment:
Any reason for picking this specific version?

Reply:
No varshaa. I added because 0.18.2 is the latest. Let me know if wanted to use range.

Reply:
I think that just depending on future without specifying a version should be sufficient.